src/lightning/fabric/plugins/precision/precision.py

```diff
@@ -97,6 +97,9 @@ def main_params(self, optimizer: Optimizer) -> _PARAMETERS:
         for group in optimizer.param_groups:
             yield from group["params"]
 
+    def unscale_gradients_(self, optimizer: Optimizer) -> None:
```

Comment:
I wouldn't add the underscore at the end. We don't follow torch's "inplace" operation notation anywhere else.